src/main/java/io/seqera/migtool/MigRecord.java

```diff
@@ -40,6 +40,16 @@ static Language from(String fileName) {
     boolean isPatch;
     boolean isOverride;
 
+    /**
```

Comment:
For example this is missing @pditommaso , otherwise it won't work from Java...

Reply:
Is this to Postgres?

Reply:
> Is this related to Postgres?

<img width="824" alt="image" src="https://github.com/user-attachments/assets/5b56dcc6-5b47-46eb-95c4-74519c3d1811">

This won't work from Java, but I guess it is not needed for Postgres.